integration/mediation-tests/tests-other/src/test/java/org/wso2/carbon/esb/synapse/common/formatters/ESBJAVA3290TestXWWWFormURLEncodedFormatter.java

```diff
@@ -26,25 +25,12 @@
 
     private static final String synapseConfig = "x_www_form_url_encoded_formatter_test.xml";
     public WireMonitorServer wireServer;
-    private ServerConfigurationManager serverConfigurationManager;
 
     @BeforeTest(alwaysRun = true)
```

Comment:
this should be BeforeClass instead of BeforeTest

Reply:
fixed